x-pack/plugins/security_solution/public/resolver/view/panel.tsx

```diff
@@ -24,7 +24,7 @@ import { useResolverDispatch } from './use_resolver_dispatch';
 import * as event from '../../../common/endpoint/models/event';
 import { ResolverEvent, ResolverNodeStats } from '../../../common/endpoint/types';
 import { SideEffectContext } from './side_effect_context';
-import { ProcessEventListNarrowedByType } from './panels/panel_content_related_list';
+import { ProcessEventListNarrowedByType } from './panels/process_event_list_narrowed_by_type';
```

Comment:
this file should probably be panels/index.tsx instead of just panel.tsx

Reply:
:red_circle: That's not a list of `process event`s - it's a list of related events like (registry creation, file creation, etc.). Also, if we're changing the name of one of the panel content components, we should change the prefix on all of them.

Reply:
I gave it the wrong name from the export -> The file name should stay the same, the export name should change,

Reply:
Maybe something like `relatedEventListNarrowedByType` ?

Reply:
I'll revert this part of the PR.